File: responses/2ZU89B-9VF/count_of_largest_integer.py
# you can write to stdout for debugging purposes, e.g.
# print("this is a debug message")

def solution(S):
    # write your code in Python
    mySum = 0
    if type(S) == str:
        for elem in S.split(','):
            mySum += int(elem)

    return mySum 

File: responses/VHXJN9-52D/count_of_largest_integer.js
// you can write to stdout for debugging purposes, e.g.
// console.log('this is a debug message');

function solution(S) {
    var numbersArray = S.split(',');
    var sum =0;
    for(var i=0; i<numbersArray.length; i++){
        sum+=parseInt(numbersArray[i]);
     }
return sum;
}